### PR TITLE
Add Los Angeles to list of meetups

### PR DESCRIPTION
**Pull request by davidguttman**
https://js.la/code-of-conduct

<3

### CODE REVIEW

File: locale/en/get-involved/node-meetups.md [hunk 1]
```diff
@@ -343,6 +343,14 @@ REQUIREMENTS
 
 #### California
 
+#### Los Angeles
+
+- [js.la](https://js.la)
+- Frequency of meetups - monthly
+- How to submit a talk? [contribute.js.la](https://contribute.js.la)
+- Organizer name - David Guttman
+- Organizer contact info -
```
Comment (ghost) [inline locale/en/get-involved/node-meetups.md]: So are you sure no contact info? I'm OK with other info :)

**Review by ghost — APPROVED** [review]

File: locale/en/get-involved/node-meetups.md (hunk 1 above)
Comment (davidguttman) [reply locale/en/get-involved/node-meetups.md]: Ok, I added my slack info which is the best way to reach me =)

File: locale/en/get-involved/node-meetups.md [hunk 2]
```diff
@@ -343,6 +343,14 @@ REQUIREMENTS
 
 #### California
 
+#### Los Angeles
```
Comment (ZYSzys) [inline locale/en/get-involved/node-meetups.md]: Nit:

```suggestion
##### Los Angeles
```

**Review by ZYSzys — APPROVED** [review]

**Comment by ZYSzys** [comment]
Thanks !

**Pull request merged**